apache-hbase-mixin/signals/cluster.libsonnet

```diff
@@ -0,0 +1,316 @@
+        name: 'Live RegionServers',
+        type: 'gauge',
+        description: 'Number of RegionServers that are currently live.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'server_num_region_servers{%(queriesSelector)s, isactivemaster="true"}',
+          },
+        },
+      },
+
+      deadRegionServers: {
+        name: 'Dead RegionServers',
+        type: 'gauge',
+        description: 'Number of RegionServers that are currently dead.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'server_num_dead_region_servers{%(queriesSelector)s, isactivemaster="true"}',
+          },
+        },
+      },
+
+      // Connections
+      masterConnections: {
+        name: 'Master connections',
+        type: 'gauge',
+        description: 'Number of open connections to the master.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_num_open_connections{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - masters',
+          },
+        },
+      },
+
+      regionServerConnections: {
+        name: 'RegionServer connections',
+        type: 'gauge',
+        description: 'Number of open connections to RegionServers.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (region_server_num_open_connections{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - RegionServers',
+          },
+        },
+      },
+
+      // Authentication
+      masterAuthenticationSuccess: {
+        name: 'Master authentication successes',
+        type: 'raw',
+        description: 'Rate of successful authentications to the master.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_authentication_successes{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - masters success',
+          },
+        },
+      },
+
+      masterAuthenticationFailure: {
+        name: 'Master authentication failures',
+        type: 'raw',
+        description: 'Rate of failed authentications to the master.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_authentication_failures{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - masters failure',
+          },
+        },
+      },
+
+      regionServerAuthenticationSuccess: {
+        name: 'RegionServer authentication successes',
+        type: 'raw',
+        description: 'Rate of successful authentications to RegionServers.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (region_server_authentication_successes{%(queriesSelector)s})',
```

Comment:
Duplicate of the same signal with the same metric (but different query) in `regionserver.libsonnet`: https://github.com/grafana/jsonnet-libs/pull/1537/files#r2560283881